pkg/converter/volume_downgrade.go

```diff
@@ -75,7 +75,7 @@ func volumeDowngradeV1Beta1ToV1alpha1(unstructured *unstructured.Unstructured) e
 		return err
 	}
 
-	klog.V(10).Infof("Converting directcsivolume: %v to v1alpha1", v1beta1DirectCSIVolume.Name)
+	klog.V(10).Infof("Converting directpvvolume: %v to v1alpha1", v1beta1DirectCSIVolume.Name)
```

Comment:
we have not yet moved to directpvvolume crd. we can call it directcsivolume only.